11.4.Вывод_элементов_списка/11.4.2.py

```diff
@@ -0,0 +1,7 @@
+numbers = [1, 78, 23, -65, 99, 9089, 34, -32, 0, -67, 1, 11, 111]
+total = 0
+
+for i in numbers:
+    total += i * i
```

Comment:
Нас просят суммировать квадраты чисел, в этом случае лучше использовать оператор возведения в степень:
```suggestion
    total += i ** 2
```